tbx/static_src/sass/components/_footer.scss

```diff
@@ -91,4 +91,55 @@
             filter: none;
         }
     }
+
+    &__carbon-impact-container {
+        background-color: var(--color--footer-background);
+        border-top: 1px solid $color--grey-80;
+    }
+
+    &__carbon-impact {
+        display: flex;
+        padding: $spacer-mini-plus 0;
+        gap: $spacer-mini-plus;
+        width: 100%;
+
+        &--mobile {
+            display: block;
+
+            @include media-query(large) {
+                display: none;
+            }
+        }
+
+        &--desktop {
+            display: none;
+
+            @include media-query(large) {
+                display: block;
+            }
+        }
+    }
+
+    &__carbon-impact-link {
+        text-decoration: underline;
```

Comment:
There is a `link-styles` mixin you can use here, but as per our slack discussion I think the link will probably be separate.